app/views/guidance/_check_data.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
We should remove the preceding underscore from the file name as we will not use this as a partial.